app/views/director/kids/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1> Edit <%= @kid.first_name %> <%= @kid.last_name %> 's Information </h1>
+
+<%= form_for @kid do |form| %>
+
+<%= form.label :first_name %>
+<%= form.text_field :first_name %>
+
+<%= form.label :last_name %>
+<%= form.text_field :last_name %>
+
+<%= form.label :birthdate %>
+<%= form.date_field :birthdate %>
+
+<%= form.label :social_insurance_number %>
```

Comment:
Does the symbol auto convert it into text? Is that why you're not using `:sin`? Maybe there's a way to set the label to `:sin`, but change the text itself?